.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: go
 
-sudo: false
```

Comment:
This is no longer needed for Travis linux infra.
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration